internal/config/config_test.go

```diff
@@ -447,33 +445,22 @@ func TestParseExpirationDate(t *testing.T) {
 }
 
 func TestExpirationDate_MarshalText(t *testing.T) {
-	tests := []struct {
-		name    string
-		date    ExpirationDate
-		want    string
-		wantErr error
-	}{
-		{
-			name:    "valid date",
-			date:    parseED("2024/07/05"),
-			want:    "2024/07/05",
-			wantErr: nil,
-		},
+	date := mustParseExpDate("2024/07/05")
+	want := "2024/07/05"
```

Comment:
I'd change this to `[]byte`. At the end of the day that's what you expect.

```suggestion
	want := []byte("2024/07/05")
```

This also removes the need for a type conversion in line 455.